.github/actions/choco/Dockerfile

```diff
@@ -7,11 +7,9 @@ FROM mono:5.20 as builder
 ARG CHOCOVERSION=0.10.16-beta
 
 ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get update && apt-get install -y wget tar gzip
 
 WORKDIR /usr/local/src
-RUN wget "https://github.com/chocolatey/choco/archive/${CHOCOVERSION}.tar.gz"
-RUN tar -xzf "${CHOCOVERSION}.tar.gz"
+RUN curl -L "https://github.com/chocolatey/choco/archive/${CHOCOVERSION}.tar.gz" | tar -xz
 RUN mv "choco-${CHOCOVERSION}" choco
```

Comment:
`wget`, `tar` and `gzip` are not that big and provide different behaviour to `busybox`, I would like to leave it like that since it works